src/states/LevelOne.js

```diff
@@ -5,12 +5,16 @@ var LevelOne = function () {
   this.returnBtn = null;
   this.sceneBtn = null;
   this.scene = null;
+  this.backgroundImg = null;
 };
 
 LevelOne.prototype = {
 
   preload: function() {
     game.stage.backgroundColor = '#444444';
+    this.backgroundImg = game.add.image(0, 0, game.cache.getBitmapData('background'));
+    this.backgroundImg.width *= game.width;
```

Comment:
This should probably be `=` instead of `*=`, right? It looks right when I run it, but it feels like its wrong.